docs/cluster.md

```diff
@@ -47,8 +47,8 @@ where
 
 ```shell
 CLUSTERX SETNODES
-  "67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1 127.0.0.1 30002 master - 5461-10922 16380 16383
-    07c37dfeb235213a872192d90877d0cd55635b91 127.0.0.1 30004 slave 67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1"
+  "67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1 127.0.0.1 30002 master - 5461-10922 16380 16383\n
```

Comment:
```suggestion
  "67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1 127.0.0.1 30002 master - 5461-10922 16380 16383 \
```

I'm unsure how redis-cli continues on lines, but may not `\n`? Also, you'd better not pull in unrelated changes in this patch.

Reply:
done